design/FY2017/NFP-UpdateLEEDSummaryReport.md

```diff
@@ -0,0 +1,1386 @@
+
+<b>Schedules-SetPoints (Schedule Type=Temperature)</b><br><br>
+<table border="1" cellpadding="4" cellspacing="0">
+  <tr><td></td>
+    <td align="right">First Object Used</td>
+    <td align="right">11am Monday [C]</td>
+    <td align="right">11pm Monday [C]</td>
+  </tr>
+  <tr>
+    <td align="right">Htg-SetP-Sch</td>
+    <td align="right">HeatingSetpoint</td>
+    <td align="right">21.1</td>
+    <td align="right">12.8</td>
+  </tr>
+  <tr>
+    <td align="right">Clg-SetP-Sch</td>
+    <td align="right">CoolingSetpoint</td>
+    <td align="right">23.9</td>
+    <td align="right">40.0</td>
+  </tr>
+</table>
+
+<b>Opaque Assemblies</b><br><br>
+<table border="1" cellpadding="4" cellspacing="0">
+  <tr><td></td>
+    <td align="right">First Object Used</td>
+    <td align="right">Assembly U-Factor with Film [W/m2-K]</td>
+    <td align="right">Reflectance</td>
+  </tr>
+  <tr>
+    <td align="right">WALL-1</td>
+    <td align="right">North WALL</td>
+    <td align="right">0.384</td>
+    <td align="right">0.22</td>
+  </tr>
+  <tr>
+    <td align="right">ROOF-1</td>
+    <td align="right">Top</td>
+    <td align="right">0.271</td>
+    <td align="right">0.35</td>
+  </tr>
+</table>
+
+<b>Shading and Fenestration - Above Grade Wall and Vertical Glazing Area</b><br><br>
+<table border="1" cellpadding="4" cellspacing="0">
+  <tr><td></td>
+    <td align="right">Above Grade Wall Area [m2]</td>
+    <td align="right">Vertical Glazing Area [m2]</td>
+    <td align="right">Above Ground Window-Wall Ratio [%]</td>
+  </tr>
+  <tr>
+    <td align="right">North (315 to 45 deg)</td>
+    <td align="right">       91.50</td>
+    <td align="right">       20.85</td>
+    <td align="right">       22.79</td>
+  </tr>
+  <tr>
+    <td align="right">East (45 to 135 deg)</td>
+    <td align="right">       45.60</td>
+    <td align="right">        9.12</td>
+    <td align="right">       20.00</td>
+  </tr>
+  <tr>
+    <td align="right">South (135 to 225 deg)</td>
+    <td align="right">       91.50</td>
+    <td align="right">       21.81</td>
+    <td align="right">       23.84</td>
+  </tr>
+  <tr>
+    <td align="right">West (225 to 315 deg)</td>
+    <td align="right">       45.60</td>
+    <td align="right">        9.12</td>
+    <td align="right">       20.00</td>
+  </tr>
+  <tr>
+    <td align="right">Total</td>
+    <td align="right">      274.20</td>
+    <td align="right">       60.90</td>
+    <td align="right">       22.21</td>
+  </tr>
+</table>
+
+<b>Shading and Fenestration - Roof and Skylight Area</b><br><br>
+<table border="1" cellpadding="4" cellspacing="0">
+  <tr><td></td>
+    <td align="right">Roof Area [m2]</td>
+    <td align="right">Skylight Area [m2]</td>
+    <td align="right">Skylight-Roof Ratio [%]</td>
+  </tr>
+  <tr>
+    <td align="right">       Total</td>
+    <td align="right">      463.60</td>
+    <td align="right">        0.00</td>
+    <td align="right">        0.00</td>
+</table>
+
+<b>Lighting - Interior</b><br><br>
```

Comment:
Yes, some of the tables are identical or very similar. One of the issues people had was how many different places they needed to look for the information. The revised report is trying to bring that all together to make filling out the LEED spreadsheet as easy as possible.